File: library/src/main/java/com/getstream/sdk/chat/logger/StreamLoggerHandler.java
package com.getstream.sdk.chat.logger;

import androidx.annotation.NonNull;

public interface StreamLoggerHandler {

    void logT(@NonNull Throwable throwable);

    void logT(@NonNull String className, @NonNull Throwable throwable);

    void logI(@NonNull String className, @NonNull String message);

    void logD(@NonNull String className, @NonNull String message);

    void logW(@NonNull String className, @NonNull String message);

    void logE(@NonNull String className, @NonNull String message);
}

File: library/src/main/java/io/getstream/chat/android/client/logger/StreamLoggerHandler.kt
package io.getstream.chat.android.client.logger

interface StreamLoggerHandler {
    fun logT(throwable: Throwable)

    fun logT(className: String, throwable: Throwable)

    fun logI(className: String, message: String)

    fun logD(className: String, message: String)

    fun logW(className: String, message: String)

    fun logE(className: String, message: String)
}